src/track/track.cpp

```diff
@@ -744,6 +744,21 @@ void Track::setCuePoint(CuePosition cue) {
     emit cuesUpdated();
 }
 
+void Track::shiftCuePositions(double milliseconds) {
+    QMutexLocker lock(&m_qMutex);
+
+    VERIFY_OR_DEBUG_ASSERT(m_streamInfo) {
+        return;
+    }
+    double frames = m_streamInfo->getSignalInfo().millis2frames(milliseconds);
+    for (const CuePointer& pCue : m_cuePoints) {
+        pCue->shiftPosition(frames);
+    }
+
+    markDirtyAndUnlock(&lock);
+    emit cuesUpdated();
```

Comment:
This is the signal when cues are added or removed. 
I think the signal from the cue itself should be sufficient.

Reply:
You're right, this signal isn't necessary.